Src/Fluent.sublime-syntax

```diff
@@ -300,7 +321,7 @@ contexts:
 
   argument-colon:
   - match: ':'
-    scope: punctuation.separator.fluent
+    scope: punctuation.separator.argument.value.fluent
```

Comment:
Oh, didn’t know about these refinements. Thanks. Wish they were mentioned in the
[official docs][subl-scopes] on scopes…

[subl-scopes]: https://www.sublimetext.com/docs/3/scope_naming.html